src/app/containers/main-error-page/main-error-page.component.html

```diff
@@ -0,0 +1,16 @@
+<exui-app-header></exui-app-header>
+<exui-page-wrapper [title]="'Error Page'">
```

Comment:
@waynepHMCTS - is it intentional to have the title showing as `'Error Page'`, with the single quote marks?

Reply:
Yes